src/invidious/routes/before_all.cr

```diff
@@ -100,6 +88,24 @@ module Invidious::Routes::BeforeAll
       end
     end
 
+    unregistered_path_whitelist = {"/", "/login", "/licenses", "/privacy"}
```

Comment:
```suggestion
    unregistered_path_whitelist = {"/", "/api/v1/comments/jNQXAC9IVRw", "/api/v1/videos/dQw4w9WgXcQ", "/login", "/licenses", "/privacy"}
```

Adding `/api/v1/comments/jNQXAC9IVRw` allows the health-check to succeed and `/api/v1/videos/dQw4w9WgXcQ` to add the instance to, e.g., Yattee.